beetsplug/discogs.py

```diff
@@ -318,7 +319,13 @@ def get_album_info(self, result):
         country = result.data.get('country')
         data_url = result.data.get('uri')
         style = self.format(result.data.get('styles'))
-        genre = self.format(result.data.get('genres'))
```

Comment:
Maybe this could be simplified a bit by keeping the genre itself in a variable, like:

```py
base_genre = self.format(result.data.get('genres'))
```

…and using that in the assignments below.